### ISSUE TITLE
Add field to json proxied, showing which its original registry was.

### ISSUE THREAD

**Issue by aredridel** [krakenjs/kappa#105]
I'd love to make sure we can tell internal from external registry information apart.

**Comment by alexindigo** [comment]
+1 :)

As workaround I'm checking `package.repository.url` for internal domain on frontend.

**Comment by aredridel** [comment]
Yeah, though that comes from the package, not the registry -- one of the uses of this change would be to identify things published to the wrong registry.

**Comment by alexindigo** [comment]
Interesting, can you elaborate on use case?

**Comment by aredridel** [comment]
We had a package that should have been published to `registry.npmjs.com` get published to our internal registry by mistake (shadowing the real package) -- diagnosing this was a little bit of guesswork since nothing outright said which registry the answer came from.

**Comment by alexindigo** [comment]
I see. We're trying to have only-local-per-package-npmrc rule to fight this kind of issues. But more visibility will definitely help.

**Comment by jasisk** [comment]
Unfortunately, this is a tough thing to accomplish given we're operating within the confines of a CLI application we don't own. That said, we do [add an `x-registry` header](https://github.com/krakenjs/kappa/blob/dd9b26ecfa02fda98e39b7a5362d7999721fd13e/lib/delegate/proxy.js#L167) to every response. I _believe_ (without looking ... fingers crossed) you can see that information if you set verbose logging.

Any ideas how else we might be able to accomplish this?

**Comment by aredridel** [comment]
Perhaps add `"kappaRegistryInfo": { "origin": "registryname" }` to the returned JSON?

**Comment by aredridel** [comment]
(and no, it doesn't show with `--verbose` to npm)

**Comment by jasisk** [comment]
Shows up with `--loglevel=silly` but, it is, after all, a silly amount of logging.

**Comment by aredridel** [comment]
That is indeed silly.
